### ISSUE TITLE
imagePullPolicy is not accurate/correct

### ISSUE THREAD

**Issue by Emilio-Pega** [pegasystems/pega-helm-charts#56]
The described `imagePullPolicy` in the readme appears to be inaccurate.  The templates (as far as I can tell) do not explicitly set `imagePullPolicy`

The readme:
![Capture](https://user-images.githubusercontent.com/54679517/71815811-a8745500-304e-11ea-960f-430bd5c07876.PNG)

The actual behavior if  `imagePullPolicy` is not specified https://kubernetes.io/docs/concepts/configuration/overview/#container-images

Basically, the readme is only accurate if you use a docker tag of `latest` or do not specify a docker tag.

**Comment by Emilio-Pega** [comment]
Please either update the documentation -or- explicitly set the `imagePullPolicy` to `Always`

**Comment by Emilio-Pega** [comment]
SQuID/Octopodes would prefer if you explicitly set `imagePullPolicy` to `Always`

**Comment by Emilio-Pega** [comment]
Internal BUG-533953 created to track
